docs/pages/access-controls/guides/hardware-key-support.mdx

```diff
@@ -77,6 +77,9 @@ like `tctl edit`. With touch required, hardware key support provides better secu
   so we recommend only using one PIV application at a time.
 </Admonition>
 
+- Running Smart Card daemon:
+  - Ships with MacOS and Windows, no install necessary.
+  - On Linux: `apt get pcscd`. Check status with `systemctl status pcscd`.
```

Comment:
`apt` doesn't work on all Linux distributions, only on Debian-based as Debian and Ubuntu. I'd mention that this is an Ubuntu-only command. I'd also add something like "consult your distribution manual on how to install pcscd" for those who are running something different.